code/controllers/subsystem/modular_computers.dm

```diff
@@ -40,41 +35,17 @@ SUBSYSTEM_DEF(modular_computers)
 		if(prog.available_on_syndinet)
 			available_antag_software.Add(prog)
 
-///Checks if at least one ntnet relay is functional.
-/datum/controller/subsystem/modular_computers/proc/check_relay_operation()
-	for(var/obj/machinery/ntnet_relay/relays as anything in ntnet_relays)
-		if(!relays.is_operational)
-			continue
-		return TRUE
-	return FALSE
-
-/datum/controller/subsystem/modular_computers/proc/toggle_function(function)
-	if(!function)
-		return
-	function = text2num(function)
-	switch(function)
-		if(NTNET_SOFTWAREDOWNLOAD)
-			setting_softwaredownload = !setting_softwaredownload
-			SSnetworks.add_log("Configuration Updated. Wireless network firewall now [setting_softwaredownload ? "allows" : "disallows"] connection to software repositories.")
-		if(NTNET_COMMUNICATION)
-			setting_communication = !setting_communication
-			SSnetworks.add_log("Configuration Updated. Wireless network firewall now [setting_communication ? "allows" : "disallows"] instant messaging and similar communication services.")
-
 ///Checks whether NTNet is available for a specific function, checking NTNet relays and shutdowns. If none is passed, none is needed.
-/datum/controller/subsystem/modular_computers/proc/check_function(specific_action = NONE)
+/datum/controller/subsystem/modular_computers/proc/check_function()
 	// No relays found. NTNet is down
 	if(!length(ntnet_relays))
 		return FALSE
 	// Check all relays. If we have at least one working relay, network is up.
-	if(!check_relay_operation())
-		return FALSE
-
-	switch(specific_action)
-		if(NTNET_SOFTWAREDOWNLOAD)
-			return setting_softwaredownload
-		if(NTNET_COMMUNICATION)
-			return setting_communication
-	return TRUE
+	for(var/obj/machinery/ntnet_relay/relays as anything in ntnet_relays)
```

Comment:
You're going to end up needing to reuse a function like this for removing NTNet from circuits (since they still need to require a relay). You probably want this as a global or something else.

Reply:
ntnet_relays are stored in this subsystem, I don't know how circuits work but I think it would be fine if they just used this

Reply:
It's very odd structurally for circuits (and other NTNet products that theroetically might exist in the future) to be depending on the modular computer subsystem. Is it difficult to make a different, new subsystem track the relays?

Reply:
My hopes were that ntos would be the replacement for ntnet, my commit to PR after this to remove [all networks ](https://github.com/JohnFulpWillard/tgstation/commit/18e52583b4b3fb0d6e6b05b49043845b916dbd6b) shows that there's only 2 circuit parts that actually use ntnet, I don't know exactly how they work so I'll look into it

Reply:
The only reason those circuit components exist is so that there's a physical machine (the relays) that can be destroyed and block their communication

Reply:
I brought the circuit back, I can move it away from SSmodular_computers when it's time to PR it- https://github.com/JohnFulpWillard/tgstation/commit/be12cbb8ddd10c950aa09b92b1b0da2f4c2c84f1